src/librustc_lint/lib.rs

```diff
@@ -190,6 +190,10 @@ pub fn register_builtins(store: &mut lint::LintStore, sess: Option<&Session>) {
             id: LintId::of(ILLEGAL_STRUCT_OR_ENUM_CONSTANT_PATTERN),
             reference: "RFC 1445 <https://github.com/rust-lang/rfcs/pull/1445>",
         },
+        FutureIncompatibleInfo {
+            id: LintId::of(UNSIZED_IN_TUPLE),
+            reference: "RFC PR 1592 <https://github.com/rust-lang/rfcs/pull/1592>",
+        }
```

Comment:
Should we add info for `OBJECT_UNSAFE_FRAGMENT` too?

Also, can you point this at the summary issue that I see you are drafting? (https://github.com/rust-lang/rust/issues/33242)